app-tasks/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN set -ex \
                --shell /usr/sbin/nologin \
                airflow \
```

Comment:
Should have mentioned it before, but there may not be much value in keeping the `airflow` user and group creation steps in here if we aren't going to use them.